hbase-rsgroup/src/test/java/org/apache/hadoop/hbase/rsgroup/TestRSGroupsKillRS.java

```diff
@@ -237,12 +238,12 @@ public void testLowerMetaGroupVersion() throws Exception{
     addGroup(groupName, groupRSCount);
 
     // move hbase:meta to meta_group
-    tableName = TableName.META_TABLE_NAME;
     Set<TableName> toAddTables = new HashSet<>();
-    toAddTables.add(tableName);
+    toAddTables.add(TableName.META_TABLE_NAME);
     rsGroupAdmin.moveTables(toAddTables, groupName);
-    assertTrue(rsGroupAdmin.getRSGroupInfo(groupName).getTables().contains(tableName));
-    TEST_UTIL.waitTableAvailable(tableName, 30000);
+    assertTrue(
+        rsGroupAdmin.getRSGroupInfo(groupName).getTables().contains(TableName.META_TABLE_NAME));
+    TEST_UTIL.waitUntilAllRegionsAssigned(TableName.META_TABLE_NAME, 30000);
```

Comment:
Does this method work for meta table? I'm not sure, just asking. As lots of the methods in HBTU just read the meta table so can not be used to test meta table itself...